source/jormungandr/jormungandr/street_network/geovelo.py

```diff
@@ -391,6 +460,20 @@ def _direct_path(
         direct_path_type,
         request_id,
     ):
+        if self.use_direct_path_service_backup(
+            pt_object_origin, pt_object_destination, fallback_extremity.represents_start
```

Comment:
use `direct_path_type` instead of `fallback_extremity`

Reply:
:+1: